source/image-handler/Dockerfile

```diff
@@ -0,0 +1,83 @@
+# Define custom function directory
+ARG FUNCTION_DIR="/function"
+
+FROM node:16-bookworm
```

Comment:
debian bookworm was the magic distro (after several attempts) that both has a sane C build environment and libs (👎 amazon linux) and a prebuilt node stack that can easily work with lambda (👎 ubuntu)